anagrafica/models.py

```diff
@@ -1219,6 +1266,121 @@ def segmenti_collegati(self):
                         attivi.append({'segmento': segmento, 'sede': sede})
         return attivi
 
+    @property
+    def regione(self):
```

Comment:
`regione` e' ambiguo nel contesto del modello Persona. E' una regione di residenza, di nascita, o di appartenenza? Persona e' un modello per le schede anagrafiche, la maggior parte delle persone in database non hanno neanche una sede di appartenenza. Se e' proprio necessario, suggerirei qualcosa come `regione_sede_riferimento`. Ma se `persona.sede_riferimento` e' `None`, questo metodo dovrebbe ritornare `None`.

Reply:
direi che la togliamo in virtù del commento precedente sul dominio. Effettivamente torno `None` se manca sede di riferimento con locazione